enshrine-esdt-safe/src/to_sovereign/create_tx.rs

```diff
@@ -60,14 +60,14 @@ pub trait CreateTxModule:
             return;
         }
 
-        let all_tokens = self.verify_items_signature(opt_signature, tokens);
-        self.burn_tokens().extend(&all_tokens);
+        // let all_tokens = self.verify_bls_signature(opt_signature, tokens);
```

Comment:
commented code

Reply:
this code will be changed once the the `feat/enshrine` branch will be merged into this one, the commented code is a really old version that will be replaced